pkg/kubectl/resource/visitor.go

```diff
@@ -154,6 +154,17 @@ func (i *Info) Refresh(obj runtime.Object, ignoreError bool) error {
 	return nil
 }
 
+// String returns the general purpose string representation
+func (i *Info) String() string {
+	basicInfo := fmt.Sprintf("Name: %q, Namespace: %q\nObject: %+q", i.Name, i.Namespace, i.Object)
+	if i.Mapping != nil {
+		mappingInfo := fmt.Sprintf("Resource: %q, Version: %q", i.Mapping.Resource,
```

Comment:
Nit: I think it's more accurate to use `GroupVersionKind` instead of `Version`.

Reply:
Thanks, fixed.